demo/src/main/java/com/eis/geoCalendar/demo/Behaviour/EventMapBehaviour.java

```diff
@@ -0,0 +1,470 @@
+     * This method allows you to define a visible region on the map, to signal to the map that portions
+     * of the map around the edges may be obscured, by setting padding on each of the four edges of the map.
+     * Map functions will be adapted to the padding. For example, the zoom controls, compass, copyright notices
+     * and Google logo will be moved to fit inside the defined region, camera movements will be relative
+     * to the center of the visible region, etc.
+     *
+     * @throws NullPointerException if the method is called before map's initialization
+     * @param left      The number of pixels of padding to be added on the left of the map. (>= 0)
+     * @param top       The number of pixels of padding to be added on the top of the map. (>= 0)
+     * @param right     The number of pixels of padding to be added on the right of the map. (>= 0)
+     * @param bottom    The number of pixels of padding to be added on the bottom of the map. (>= 0)
+     */
+    public void setMapPadding(int left, int top, int right, int bottom) {
+        mMap.setPadding(left, top, right, bottom);
+    }
+
+    /**
+     * These events are defined elsewhere, so no need to notify listeners
+     *
+     * @param events A bunch of events to position in the map (both description and Position must be defined)
+     */
+    public void addEventsToMap(List<Event<String>> events) {
+        for (Event<String> event : events) {
+            Marker created = mMap.addMarker(new MarkerOptions().position(new LatLng(event.getPosition().getLatitude(),
+                    event.getPosition().getLongitude()))
+                    .title(event.getContent()));
+            created.setTag(event);
+        }
+    }
+
+
+    /**
+     * Called when user clicks (taps) on the map for a prolonged time
+     * Collects the position associated and opens Dialog for creating event
+     *
+     * @param latLng The position on the map where the user clicked
+     */
+    @Override
+    public void onMapLongClick(LatLng latLng) {
+        addEventDialog.setEventPosition(latLng);
+        addEventDialog.show(supportFragmentManager, CREATE_EVENT_DIALOG_TAG);
+    }
+
+    /**
+     * Moves the camera to the marker's location and, if set,
+     * expands the bottomSheet to display Marker's event's description
+     * If a goToNavigatorView was set, it will turn visible
+     * Note: returning false for this event will fire the default event for Google Map
+     *
+     * @param marker The marker that caused this callback
+     * @return True if the listener has consumed the event, false otherwise.
+     */
+    @Override
+    public boolean onMarkerClick(Marker marker) {
+        currentFocusMarker = marker;
+        moveMap(marker.getPosition());
+
+        if (goToNavigatorView != null)
+            goToNavigatorView.setVisibility(View.VISIBLE);
+
+        if (bottomSheetBehaviour != null) {
+            bottomSheetBehaviour.setDisplayedText(marker.getTitle());

```

Comment:
Same thing as the previous comment review
```suggestion
            bottomSheetBehaviour.setDisplayedText(currentFocusMarker.getTitle());

```